src/ngraph/runtime/interpreter/int_backend.cpp

```diff
@@ -63,6 +64,7 @@ bool runtime::interpreter::INTBackend::compile(shared_ptr<Function> function)
     {
         instance.m_is_compiled = true;
         pass::Manager pass_manager;
+        pass_manager.register_pass<pass::NopElimination>();
```

Comment:
okay, adding optimization passes to Interpreter which is supposed to be our reference backend is officially scary.

Reply:
Agreed. Any comment on why we need this?

Reply:
We need to convert the constant and broadcast that take their shape from another op back into regular constant and broadcast.